__tests__/index.js

```diff
@@ -25,6 +25,8 @@ pluginTester({
     'static import (with file extension)': 'import("./Foo.js")',
     'static import (string template)': 'import(`./base`)',
     'static import (string template + relative paths)': 'import(`../../base`)',
+    'static import (with relative paths + nested folder)':
+      'const obj = {component:()=>import(`../components/nestedComponent`)}; ()=> obj.component()',
```

Comment:
So why did i make an object and a function?

Its hella hard to resolve UniversalComponent with strings placed elsewhere. I initially had one util file per folder, but it would create risk among a big company, people might import the wrong thing. 

```
<UniversalComponent component={()=>import('../components/generalUI/Slider')}/> // now it resolves from this file, keeping with the common sense ES6 way of doing things. Devs just work normally ```

Reply:
Hey what's goin on brother, i dont know if im losing my mind or just havent been in this code in a while, but is `<UniversalComponent />` your own HoC--cuz `import()` usually occurs before *render time*.

Also, what's the main problem we're trying to solve--that automatic chunk name creation requires the same initial folder name (in this case `/components/`) in order to generate the same chunk name for identical imports from different places in the file system?

Reply:
😂 yeah it’s your universal HOC, but my component which is a little utility. 

Pretty much the problem is that the way you demonstrated works just fine as a string. But I’ve got many files. So when I need a component, I must supply a path relative to where my utility is. That’s way I can use code completion and can also do reverse dependency lookups. It removes the manual labor of having to remember where the utility is. In the example, I’m pretty much testing import as a prop rather than string. I’ll send you a snippet so you can see what the hell im talking about. In my next PR I’ll include a test that looks more normal as well for testing nested imports. 


Sorry I know it’s confusing outta context, I’ll send you how I’m using it and I’m always open to a better way